cmd/gctcli/orderbook.go

```diff
@@ -617,13 +611,7 @@ func getOrderbookStream(c *cli.Context) error {
 
 		bidLen := int64(len(resp.Bids) - 1)
 		askLen := int64(len(resp.Asks) - 1)
-
-		var maxLen int64
-		if bidLen >= askLen {
-			maxLen = bidLen
-		} else {
-			maxLen = askLen
-		}
+		maxLen := max(bidLen, askLen)
```

Comment:
If above applies, then here too.
Can provide patch if desired.

Reply:
I agree and thanks for the offer but I can do 👍

Reply:
Pushed: https://github.com/thrasher-corp/gocryptotrader/pull/1826/commits/c63c2576c7173b3e7fda0d7cb1bd10ea85b06713